misc/keygen/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "keygen"
+version = "0.1.0"
+edition = "2021"
+authors = ["demfabris <demfabris@gmail.com>"]
+license = "MIT"
+repository = "https://github.com/libp2p/rust-libp2p"
+keywords = ["peer-to-peer", "libp2p", "networking"]
+categories = ["network-programming", "asynchronous"]
+
+[dependencies]
+structopt = "0.3.26"
+zeroize = "1.5.2"
+serde = { version = "1.0.136", features = ["derive"] }
+serde_json = "1.0.79"
+libp2p = { version = "0.42.2", default-features = false }
+base64 = "0.13.0"
+num_cpus = "1.13.1"
```

Comment:
What do you think of using https://doc.rust-lang.org/std/thread/fn.available_parallelism.html ?

Reply:
Nice @mxinden! I just learned about this new addition earlier today xD.

I'm on a busy week but will soon tackle the pending stuff